app/helpers/application_helper.rb

```diff
@@ -8,6 +8,14 @@ def app_name
     #NUCore.app_name
   end
 
+  def app_name_link_text
```

Comment:
Probably. Although, I'd almost consider dropping `app_admin_name` altogether and just always say "NU Core" in the header. 

@nutodd, what do you think?

We noticed some inconsistencies with how senior staff was treated. They were not considered `operator?`, and actually neither were global admins. The only place this method is used is to determine whether to display "NU Core" or "NU Core Admin" in the header. I didn't even realize that was a thing since I'm 99% of the time logged in as an admin. Do you think we could drop the logic?